src/client/debugger/jupyter/debuggingCellMap.ts

```diff
@@ -0,0 +1,41 @@
+import { injectable } from 'inversify';
+import { NotebookCell, NotebookDocument, Uri } from 'vscode';
+import { noop } from '../../common/utils/misc';
+import { ICell, IDebuggingCellMap, INotebookExecutionLogger } from '../../datascience/types';
+
+@injectable()
+export class DebuggingCellMap implements IDebuggingCellMap, INotebookExecutionLogger {
```

Comment:
Just a draft here, but this class could use some class comments about what it's responsible for.